discovery/consul/consul.go

```diff
@@ -560,6 +583,30 @@ func (srv *consulService) watch(ctx context.Context, ch chan<- []*targetgroup.Gr
 			labels[taggedAddressesLabel+model.LabelName(name)] = model.LabelValue(v)
 		}
 
+		if serviceNode.Service.Kind != consul.ServiceKindTypical { // to avoid setting the label to an empty value
```

Comment:
It is fine to set the label to an empty value.

Reply:
ok!